.github/workflows/cooker-selfhosted.yml

```diff
@@ -29,7 +29,7 @@ on:
 jobs:
 
   Building_RetroDECK:
-    runs-on: self-hosted
+    runs-on: self-hosted, ubuntu-20.04
```

Comment:
I think this means: "a runner that contains the tag self-hosted AND ubuntu-20.04" so basically I expect that it's not running.

To do this we have to setup a runner group but GitHub hosted runners cannot be placed into groups, only self-hosted.